shunting-yard-replicator/src/main/java/com/expediagroup/shuntingyard/replicator/exec/conf/OrphanedDataStrategyConfig.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2016-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.shuntingyard.replicator.exec.conf;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import com.hotels.bdp.circustrain.api.conf.OrphanedDataStrategy;
+
+@Configuration
+@ConfigurationProperties
+public class OrphanedDataStrategyConfig {
```

Comment:
OK, so we need a suffix on this one since it's not the actual `OrphanedDataStrategy`. I see we have one other class in this package called `EventReceiverConfiguration` - should we follow the naming convention and change `Config`->`Configuration`?

Reply:
Yes good point, one day I'll name these config classes appropriately...